.moai/specs/SPEC-CRON-PATTERN-001/plan.md

```
@@ -0,0 +1,231 @@
+- [ ] **P5 Memory Hygiene Sweep**:
+  - Schedule: `0 3 * * 0`
+  - Prompt: "Audit `~/.claude/projects/<hash>/memory/MEMORY.md`. Identify entries older than 30 days that have not been referenced. Propose archive."
+  - Output: `.moai/routines/history/memory-sweep-YYYY-MM-DD.jsonl`
+  - Rationale: MEMORY 신선도 + 토큰 비용 감소
+
+**Exit Criteria**: 5 패턴 모두 4-field 명시 (schedule + prompt + output + rationale)
+
+### M3 — Plan-Tier Limits + Failure Policy (Priority: High)
+
+- [ ] Plan-Tier Limits Table:
+  | Plan | Routines/day | Recommended Patterns |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)